inst/examples/dashboard/app.R

```diff
@@ -13,8 +14,8 @@ source("home.R")
 source("about.R")
 source("footer.R")
 
-examplePages <- imap(examples, function(example, name) {
-  page <- makeExamplePage(name, example$ui)
+examplePages <- imap(examples, function(item, name) {
```

Comment:
utils::example is taken

Reply:
Ok, personally I like to avoid masking, but I understand the rationale and will change it